analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -236,46 +246,27 @@ class GoMod(
     private fun getModuleInfos(projectDir: File): Map<String, ModuleInfo> {
         val list = runGo("list", "-m", "-json", "-buildvcs=false", "all", workingDir = projectDir)
 
-        val moduleInfos = list.stdout.byteInputStream().use { JSON.decodeToSequence<ModuleInfo>(it) }
-
-        return buildMap {
-            moduleInfos.forEach { moduleInfo ->
-                if (moduleInfo.replace != null) {
-                    // The `replace` object in the output of `go list` does not have the `indirect` flag, so copy it
-                    // from the replaced module.
-                    val replace = moduleInfo.replace.copy(indirect = moduleInfo.indirect)
-                    put(moduleInfo.path, replace)
-                    put(moduleInfo.replace.path, replace)
-                } else {
-                    put(moduleInfo.path, moduleInfo)
-                }
-            }
-        }
+        return list.stdout.byteInputStream().use { JSON.decodeToSequence<ModuleInfo>(it) }.associateBy { it.path }
```

Comment:
This is a bit long for my taste. Would it make sense to keep the original `moduleInfos` (which also slightly reduces the diff), and just `return moduleInfos.associateBy { it.path }` for clarity *what* is being associated?